ql/src/java/org/apache/hadoop/hive/ql/parse/AlterTableMetaRefSpec.java

```diff
@@ -48,17 +57,17 @@ public String toString() {
         .add("operationParams", operationParams).toString();
   }
 
-  public static class CreateBranchSpec {
+  public static class CreateMetaRefSpec {
```

Comment:
in iceberg, it's called SnapshotRef

Reply:
> in iceberg project, it's called SnapshotRef, MetaRef is too wide

Renamed MetaRef to SnapshotRef.

> could we create also builders for tag and branch similar to SnapshotRef.tagBuilder(snapshotId).build();

Maybe i misunderstand what your meaning, and i think no need to create builder like iceberg project. 

`Builder `in iceberg SnapshotRef is used to create multiple constructor. e.g.  `SnapshotRef.tagBuilder(snapshotId).build()` is used to create a tag without any properties(maxRefAgeMs, minSnapshotsToKeep), `SnapshotRef.tagBuilder(1L).maxRefAgeMs(10L).build()` is used to create a tag with property `maxRefAgeMs`. 

But what we need here is just only one constructor with all properties(maxRefAgeMs, minSnapshotsToKeep..), and pass these properties to the final branch& tag creation stage,` IcebergBranchExec.java` &  `IcebergTagExec.java`, and set these properties when invoking iceberg native api to create banch & tag.

Reply:
makes sence